libcxx/include/__type_traits/datasizeof.h

```diff
@@ -26,39 +26,38 @@
 
 _LIBCPP_BEGIN_NAMESPACE_STD
 
-template <class _Tp>
-struct __libcpp_datasizeof {
 #if __has_extension(datasizeof)
-  static const size_t value = __datasizeof(_Tp);
+template <class _Tp>
+inline const size_t __datasizeof_v = __datasizeof(_Tp);
```

Comment:
What about the cost to the linker here? 

This is now external linkage with ODR deduplication?

Are we sure we don't ODR use this anywhere?

Reply:
It's exactly as much ODR-used as `__libcpp_datasizeof<>::value`. I don't see where the difference is.